app/commands/v2/put_content.rb

```diff
@@ -221,6 +222,14 @@ def update_content_item(content_item)
         content_item.save!
       end
 
+      def update_last_edited_at_if_needed(content_item, last_edited_at = nil)
+        if last_edited_at.nil? && %w(major minor).include?(payload[:update_type])
+          last_edited_at = Time.zone.now
```

Comment:
I still like just having one if, as I think it reflects the way in which I think about the condition. I understand the desired behavior as "set last_edited_at to the current time if it has not been specified in the request and the update_type is major or minor", and so having an if statement that is similar to this is quite helpful. While the alternatives above are tidier, I think they are all harder to read than just having a big if statement.

What does everyone think? I'm happy to defer to your better judgement, as this could just be an issue with my inexperience with reading Ruby?